go-selinux/selinux.go

```diff
@@ -213,6 +213,11 @@ func ReserveLabel(label string) {
 	reserveLabel(label)
 }
 
+// IsMLSEnabled checks if MLS is enabled.
+func IsMLSEnabled() bool {
```

Comment:
It's more conventional in Go to omit `Is` prefixes (as the `bool` return already describes it's a boolean result). (Same for `Get` (and `Set`), but that's already in the code, so we can't reverse that 😂).

Perhaps (same for the non-exported variants);

```go
// MLSEnabled checks if MLS is enabled.
func MLSEnabled() bool {
```

Which will be called as `selinux.MLSEnabled()`, e.g.

```go
if selinux.MLSEnabled() {
    // do magic stuff!
}
```